### ISSUE TITLE
Releasing database of openframework.stanford.edu

### ISSUE THREAD

**Issue by jagf** [SU-SWS/open_framework#260]
Hi Megan & Brian,
I have been experimenting with open_framework-7.x-2.06. I am wondering if you would consider releasing a "Distribution" copy of the openframework.stanford.edu site. This could be beneficial for users to see how your site is constructed.

Regards,
Gary
Melbourne

**Comment by hyperboy** [comment]
Thanks for the distribution suggestion.  It would probably be helpful to the users if we created some sample landing, announcements, and content pages.  We'll look into the effort required.

**Comment by MariusE** [comment]
+1

**Comment by meganem** [comment]
I don't think creating a distro or releasing our database is something we can easily, but we do give regular presentations and write a LOT of blog posts on our site building practices here at SWS. Check out the blog: https://swsblog.stanford.edu and our list of modules we work from: https://itservices.stanford.edu/service/web/stanfordsites/modules
These are basically ingredients in all our "recipes" for site building.

And the OFW documentation for tips: https://openframework.stanford.edu/guide/creating-responsive-layouts

Basically, these are the key modules:
Context
Block Class

Hope this helps!
